globals.go

```diff
@@ -115,6 +115,7 @@ const (
 	PatchStarted     = "started"
 	PatchSucceeded   = "succeeded"
 	PatchFailed      = "failed"
+	PatchAborted     = "aborted"
```

Comment:
@malikchaya2 marking you for visibility; will this conflict with your PR? I'm thinking no since it's not a status we save but we may still want to add it in . Not sure who is going to end up having to deal with the merge conflict though 😅

Reply:
I don't think this will cause a problem because this is only used by the resolver, so no patch in the db will have this status. 
The child patches are retrieved and the collective status is set before the tasks are filtered for aborted. (@khelif96  let's make sure to maintain that order when refactoring based on our offline discussion).

Reply:
Can we leave a comment here @khelif96 so that engineers in the future are aware that this is a display-only status?